### DISCUSSION TITLE
how to properly design a Showcase

### DISCUSSION THREAD

**Discussion by KotikLina** [TomSchimansky/CustomTkinter#1890] [Q&A] [answered]
I made something and would like to show it for you to look at and suggest what can be corrected or changed. how to properly format Showcase?

**Comment by AlbertSolomon** [comment, marked as answer]
To initiate a showcase on GitHub, create a new discussion, select the "showcase" label, include project screenshots within the text prompt (drag and drop images directly), and then publish.
